internal/service/mwaa/environment.go

```diff
@@ -15,12 +18,16 @@ import (
 	"github.com/hashicorp/terraform-provider-aws/internal/verify"
 )
 
+const (
+	validationRetryTimeout = 30 * time.Second
+)
+
 func ResourceEnvironment() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceEnvironmentCreate,
-		Read:   resourceEnvironmentRead,
-		Update: resourceEnvironmentUpdate,
-		Delete: resourceEnvironmentDelete,
+		CreateContext: resourceEnvironmentCreate,
+		ReadContext:   resourceEnvironmentRead,
+		UpdateContext: resourceEnvironmentUpdate,
+		DeleteContext: resourceEnvironmentDelete,
```

Comment:
Please use `WithoutTimeout` variants instead of `Context` variants - See https://github.com/hashicorp/terraform-provider-aws/issues/4177#issuecomment-696116286 and https://github.com/hashicorp/terraform-provider-aws/issues/15090.

Reply:
Updated.